skrifa/src/outline/glyf/mod.rs

```diff
@@ -741,26 +758,26 @@ where
         if have_deltas {
             self.component_delta_count = delta_base;
         }
-        if self.is_hinted {
+        if let (Some(hinter), true) = (self.hinter.as_ref(), self.is_hinted) {
```

Comment:
let chains will be nice for this stuff, e.g. `if let Some(hinter) = self.hinter && self.is_hinted { .. }`

Reply:
yeah, I occasionally attempt it and am still somehow surprised when it doesn’t work